toolchain/check/testdata/impl/fail_impl_as_scope.carbon

```diff
@@ -19,6 +19,12 @@ impl as Simple {
   fn F() {}
 }
 
+fn Function() {
+  impl as Simple {
+    fn F() {}
+  }
+}
```

Comment:
Seems weird to put this in a `fail` test

Reply:
Looks like we were suppressing the error here due to the previous error. I've split the test and we now produce an error on this like we should.